### ISSUE TITLE
Error Calling Client - IndexError: No definition '{http://service.sunat.gob.pe}billService' in 'port_types' found

### ISSUE THREAD

**Issue by samemotion** [mvantellingen/python-zeep#839]
Hi! 

1. The version of zeep. 
I have installed zeep using pip in python3.6 , using pip3 install zeep. this was done today.
zeep version.

Name: zeep
Version: 3.1.0
Summary: A modern/fast Python SOAP client based on lxml / requests
Home-page: http://docs.python-zeep.org
Author: Michael van Tellingen
Author-email: michaelvantellingen@gmail.com
License: MIT
Location: /home/samemotion/.local/lib/python3.6/site-packages
Requires: appdirs, cached-property, attrs, isodate, defusedxml, requests, six, requests-toolbelt, pytz, lxml

2. We are trying to access the following WSDL 
https://e-factura.sunat.gob.pe/ol-ti-itcpfegem/billService?wsdl

3. I am running the following code.
-------------------
from zeep import Clientt 
from zeep.wsse.username import UsernameToken
url="https://e-factura.sunat.gob.pe/ol-ti-itcpfegem/billService?wsdl"
user="user"
password="password"
client = Client(url, wsse=UsernameToken(user, password))
---------

The error message appears after running the last line

IndexError: No definition '{http://service.sunat.gob.pe}billService' in 'port_types' found

Running 
client = Client(url) 
gives the same error, so the issue should not be related  with the authentication.

Thanks a lot in advance for your help.

**Comment by mvantellingen** [comment]
I'm closing this issue due to the low quality of the initial report. You are welcome to create a new issue but please make sure it contains all the relevant information and that it is properly formatted.

Also please remember that this project is opensource and completely unpaid work.
